android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -236,6 +242,34 @@ class Screen constructor(context: ReactContext?) : FabricEnabledViewGroup(contex
             mNativeBackButtonDismissalEnabled = enableNativeBackButtonDismissal
         }
 
+    fun recalculateHeaderHeight() {
+        val summarizedHeight = when (stackPresentation) {
+            StackPresentation.TRANSPARENT_MODAL -> 0
```

Comment:
I believe transparent modals both on Android and iOS have headers and yielding `0` isn't correct